### ISSUE TITLE
napalm referencing attribute that is deprecated in py3.6 (maybe py3)

### ISSUE THREAD

**Issue by sdanelli** [napalm-automation/napalm#652]
When `ConnectTimeoutError` is attempted in "napalm/napalm/junos/junos.py" line 109, it references an attribute called `message` which has been deprecated. 

In python2.7:
```
In [1]: e = Exception('boo')

In [2]: e.message
/Users/sdanelli/.virtualenvs/containermanager/bin/ipython:1: DeprecationWarning: BaseException.message has been deprecated as of Python 2.6
  #!/Users/sdanelli/.virtualenvs/containermanager/bin/python
Out[2]: 'boo'
```

In python3.6:

```
In [6]: e = Exception('boo')

In [7]: e.message
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-7e9b9f9f4c6d> in <module>()
----> 1 e.message

AttributeError: 'Exception' object has no attribute 'message'
```
Which means is raising an `AttributeError` instead of a `ConnectionException`:

More details:
```
...
    device.open()
  File "/Users/sdanelli/devwork/napalm/napalm/junos/junos.py", line 109, in open
    raise ConnectionException(cte.message)
AttributeError: 'ConnectTimeoutError' object has no attribute 'message'
```

**Comment by sdanelli** [comment]
fix might to just cast to str if interested in the details: `raise ConnectionException(str(cte))`

**Comment by dbarrosop** [comment]
Good catch, would you mind opening a PR yourself?

**Comment by sdanelli** [comment]
@dbarrosop sure, #657

**Comment by ktbyers** [comment]
Updated fix here:  https://github.com/napalm-automation/napalm/pull/726

**Comment by dbarrosop** [comment]
Fixed via #726
